CHANGELOG.md

```diff
@@ -95,12 +95,6 @@ This is short list of the most important known issues. The full list can be retr
 - Corrupt dump files when interrupted during writing (#870).
 
 
-3.0.2 (2021-09-10)
```

Comment:
does it make sense to remove this?

Reply:
It's a duplicate/leftover. The correct 3.0.2 entry is much longer.